convertors/arachni.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+
+import os
+import json
+import argparse
+import sys
+import textwrap
+import re
+from os import path
+from collections import OrderedDict
+
+
+DEFAULT_EFFORT = 50
+
+
+def slugify(filename):
+    filename = re.sub(r'\W+', '-', filename.lower())
+    return filename.strip("-")
+
+
+def convert_multi_string(text):
```

Comment:
Yes, It was there. Default behavior is very strict, but all these options "expand_tabs=False, break_on_hyphens=False, replace_whitespace=False, break_long_words=False"  help me =)